src/components/Modals/ConfirmTransaction/components/InsufficientFunds/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import 'styled-components/macro';
+import { colors, GU } from 'components/theme';
+
+import { ReactComponent as CloseSvg } from 'assets/icons/close.svg';
+import { ReactComponent as PiggyBank } from 'assets/icons/piggy_bank.svg';
+
+import { StyledCloseContainer, StyledModalContainer, StyledPiggyBankContainer } from '../../components';
```

Comment:
I see we're using absolute imports elsewhere—should we do so here as well?

Reply:
I had it relative for fairly nested components, but yeah may as well keep it consistent: Fixed in d1b2ddbe77ae21483d3b5bee3f2ab397753e27bd